### PR TITLE
Sidebar remade

### PR DESCRIPTION
**Pull request by callumjt**
completely remade sidebar
image

changes:
- new buttons for account settings
- cleaned the code
- put the code in a folder 
- added icons

known bugs that i have no clue of fixing:
- on small screens the mail and folder buttons are off the sidebar

### CODE REVIEW

File: sidebar/script.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        search.classList = "input st";
+        search.addEventListener("keypress", function(event) {
+            if (event.key === "Enter") {
+                event.preventDefault();
+                search();
+
+                function search() {
+                    window.location.href = `https://scratch.mit.edu/search/projects?q=${
+                        document.querySelector("input.input.st").value
+                    }`;
```
[the remark this answers is not in the document]
Comment (callumjt) [reply sidebar/script.js:102]: dunno why thats bad

**Review by rgantzos — CHANGES_REQUESTED** [review]
The sidebar should use <a> elements for links, not JavaScript.

**Comment by callumjt** [comment]
> The sidebar should use elements for links, not JavaScript.

oh ok ill work on that later

**Comment by callumjt** [comment]
@Wilamaxin has made a better version cause i made this terribly lol

**Pull request closed without merging**